### PR TITLE
FEAT: Added Button component

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-12 15-08-48

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+span {
+    font-weight: bolder;
+}
+
+.btn {
+    background-color: #201D4F;
+    padding: 0.8rem;
+    color: #FFFFFF;
```
Comment (viru2001) [inline Components/Button/button.css]: you can declare variables for colors

File: Components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
+.button-section {
+    margin-top: 115rem;
+}
```
Comment (viru2001) [inline Components/Button/button.css]: why you need this big margin?

File: Components/Button/button.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    <button class="btn btn-secondary">SECONDARY</button>
+
+                    <button class="btn btn-error">ERROR</button>
+
+                    <button class="btn btn-success">SUCCESS</button>
+                </div>
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28184%2C176%2C141%2C0%29&t=3024-night&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2522%253EDEFAULT%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EPRIMARY%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253ESECONDARY%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-error%2522%253EERROR%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-success%2522%253ESUCCESS%253C%252Fbutton%253E"
+                        style="width: 589px; height: 353px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (viru2001) [inline Components/Button/button.html:85]: you should avoid using inline styles

**Review by viru2001 — COMMENTED** [review]
Some suggestions from my side : 
1. You should declare variables for colors so that you can reuse them
2. you can define variables for font size instead of using medium, small and large 
3. you can also define variables for font-weight

File: Components/Button/button.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/Button/button.css]: If I did't give this margin the sestion is going up and disappearing, while adding on content to the sestion 😢

File: Components/Button/button.html (hunk 3 above)
Comment (kotesh-arya) [reply Components/Button/button.html:85]: This is the default style from the iframe itself, should I have to change that ?

**Review by ayush65 — APPROVED** [review]

File: Components/Button/button.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+            <div class="section-footer">
+                <a href="/Components/Badge/badge.html">
+                    < Badge</a> <a href="/Components/Card/card.html"> Card >
+                </a>
+            </div>
+        </section>
+    </main>
+</body>
+
+</html>
```
Comment (Adarsh-Patel-Dev) [inline Components/Button/button.html:173]: looks good to me
Can be merged

File: Components/Button/button.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+ --default-btn-bg:#201D4F;
+}
+
+.button-section {
+    margin-top: 125rem;
+}
+
+span {
+    font-weight: var(--bolder-font-weight);
+}
```
Comment (smilegupta) [inline Components/Button/button.css:14]: why adding css to directly element?

File: Components/Button/button.css [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+.btn-success {
+    background-color: var(--alert-success);
+
+}
+
+.outline {
+    background-color: transparent;
+    color: var(--default-btn-bg);
+    border: 1px solid var(--default-btn-bg);
+}
```
Comment (smilegupta) [inline Components/Button/button.css:48]: is it talking about .outline btn? if yes name it as outline button

**Review by smilegupta — COMMENTED** [review]
LGTM! Added a few commnets

File: Components/Button/button.css (hunk 5 above)
Comment (kotesh-arya) [reply Components/Button/button.css:14]: Now I have updated it by giving that element a Class 👍🏻

File: Components/Button/button.css (hunk 6 above)
Comment (kotesh-arya) [reply Components/Button/button.css:48]: yes! , Will name it meaningfully. Thanks for pointing out 👍🏻

**Pull request merged**